src/main/java/io/confluent/examples/streams/microservices/OrderDetailsService.java

```diff
@@ -123,7 +139,10 @@ private void startConsumer(String bootstrapServers) {
     consumerConfig.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
     consumerConfig.put(ConsumerConfig.GROUP_ID_CONFIG, CONSUMER_GROUP_ID);
     consumerConfig.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-    consumerConfig.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "false");
+    final String autoCommitEnabled = eosEnabled ? "false" : "true";
```

Comment:
nit: `final String autoCommitEnabled = !eosEnabled;` (maybe even remove `autoCommitEnabled` variable and use `!eosEnabled` in next line directly?

Reply:
Done